servers/republik/modules/crowdfundings/lib/payments/stripe/deleteCustomer.js

```diff
@@ -0,0 +1,32 @@
+const getClients = require('./clients')
+
+module.exports = async ({
+  userId,
+  pgdb
+}) => {
+  const { accounts } = await getClients(pgdb)
```

Comment:
Maybe for the purpose of this script, name `accounts` to `stripeAccounts`.

Reply:
Hm, we are in stripe scope here and what other accounts should it be? ;)